sdk/digitaltwins/test-resources.json

```diff
@@ -5,7 +5,7 @@
     "_generator": {
       "name": "bicep",
       "version": "0.4.63.48766",
-      "templateHash": "14469093771711807520"
+      "templateHash": "7080001263714194046"
```

Comment:
If they have a bicep file, should the JSON file even be checked in? Unless used for something else in their workflow, it's otherwise redundant.

Reply:
We have a dev side setup script that runs the ARM template, but we can use the same bicep builder to generate it and wipe it out afterwards, I will amend that change after this is merged in.

Reply:
If it's getting used, not a big deal. Just wanted to make sure it wasn't committed in err.

Reply:
@azabbasi the existing `az` and powershell commands to deploy templates can take a bicep template as input, if your dev side scripts use those. The only thing missing is build+local validation.